app/views/users/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+
+<% @users.each do |user| %>
+  <div class='user-card'>
+    <div class='img container' >
+    <img src ='<%= user.photo %>' alt='user_image'  />
+    </div>
+    <div class='user-card img'>
+      <h2>Name: <%= user.name %></h2>
+      <h3 class='user-container'>Number of posts:<%= user.posts_counter %></h3>
+       <button>
+          <a href='/users/<%= user.id %>'> User Details </a>
+       </button>
+    </div>
+  </div>
+<% end %>
+
```

Comment:
[OPTIONAL]:
- [ ] I recommend below a better practice for displaying the users.
```ruby
   # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

Reply:
Already implemented it! Thanks for the feedback! smile